src/main/java/org/broadinstitute/hellbender/tools/spark/sv/GATKSVVCFHeaderLines.java

```diff
@@ -31,9 +31,13 @@
     public static final String HOMOLOGY_LENGTH = "HOMOLOGY_LENGTH";
     public static final String INV33 = "INV33";
     public static final String INV55 = "INV55";
-    public static final String DUPLICATED_SEQUENCE = "DUPLICATED_SEQUENCE";
+//    public static final String DUPLICATED_SEQUENCE = "DUPLICATED_SEQUENCE";
+    public static final String DUP_REPET_UNIT_REF_SPAN = "DUP_REPET_UNIT_REF_SPAN";
+    public static final String DUP_ASM_CTG_CIGARS = "DUP_ASM_CTG_CIGARS";
     public static final String DUPLICATION_NUMBERS = "DUP_NUM";
 
+    public static final String DUP_ANNOT_FROM_OPT = "DUP_ANNOT_FROM_OPT";
```

Comment:
It looks like we're always setting this, do we really need it? What if you just put an extra caveat in the header line descriptions? If you want to keep it, I might change the name to something more general, like "DUP_ANNOTATIONS_ESTIMATED" or "DUP_ANNOTATIONS_IMPRECISE".